code/game/objects/items/weapons/storage/boxes.dm

```diff
@@ -294,6 +294,16 @@
 	for(var/i in 1 to 5)
 		new /obj/item/weapon/grenade/anti_photon(src)
 
+/obj/item/weapon/storage/box/incendiary
+	name = "box of incendiary grenades"
+	desc = "A box containing 5 incendiary grenades."
+	icon_state = "box_security"
+	illustration = "flashbang"
+
+/obj/item/weapon/storage/box/incendiary/populate_contents()
+	for(var/i in 1 to 5)
+		new /obj/item/weapon/grenade/chem_grenade/incendiary(src)
```

Comment:
This is also unused, though it could stay here as something for the future.

Reply:
Leave for future, when this will be needed.